__tests__/gui/data_mapper/data_mapper.js

```diff
@@ -50,17 +50,18 @@ When(/^I click on "([^"]*)" in Data Mapper$/, function (word) {
 });
 
 Then('I select an indicator', () => {
-    cy.get('.data-category__h1_content--v2').contains('Population (2016 Community Survey)').click();
-    cy.get('.data-category__h2_content--v2').contains('Population group').click();
-    cy.get('.data-category__h3_content--v2').contains('Black african').click();
+  cy.get('.indicator-subcategory').contains('Population (2016 Community Survey)').click();
+  cy.get('.indicator-item').contains('Population group').click();
+  cy.get('.subIndicator-item').contains('Black african').click();
+  cy.get('.indicator-subcategory').contains('Population (2016 Community Survey)').click();
 
     cy.get(`${mapBottomItems} .map-options .map-options__legend`).should('be.visible');
 })
 
 When('I select another indicator', () => {
-    cy.get('.data-category__h1_content--v2').contains('Population (2011 Census)').click();
-    cy.get('.data-category__h2_content--v2').contains('Population group 2').click();
-    cy.get('.data-category__h3_content--v2').contains('Indian or Asian').click();
+  cy.get('.indicator-subcategory', {timeout: 20000}).contains('Population (2011 Census)').click();
+  cy.get('.indicator-item', {timeout: 20000}).contains('Population group 2').click();
+  cy.get('.subIndicator-item', {timeout: 20000}).contains('Indian or Asian').click();
```

Comment:
Seems like this timeout is to wait for UI to be ready after the action, can this rather be set as a default

Reply:
Thanks for noticing that @michaelglenister 
We do need timeouts now 👍

Reply:
Yes it's possible 
We can set config for cypress.
But there are different timeouts on app yet so was not sure how it would affect while running tests

Reply:
we shouldn't set 20s as the default timeout. default is 4s and it should 99% be sufficient